### PR TITLE
Fix a few style error on All teams page [SCI-1634]

### PR DESCRIPTION
**Pull request by mlorb**
[SCI-1634](https://biosistemika.atlassian.net/browse/SCI-1634)

### CODE REVIEW

File: app/javascript/src/scenes/SettingsPage/scenes/teams/components/TeamsDataTable.jsx [hunk 1]
```diff
@@ -44,47 +44,61 @@ class TeamsDataTable extends Component {
     );
   }
 
+  renderShowsTotal(start, to, total) {
+    return (
+      <span>
+        Showing { start } to { to } of { total } entries.
```
Comment (ZmagoD) [inline app/javascript/src/scenes/SettingsPage/scenes/teams/components/TeamsDataTable.jsx]: Please put this string in app/javascript/src/config/locales/messages.js and use FormatedMessage component

File: app/javascript/src/scenes/SettingsPage/scenes/teams/components/TeamsDataTable.jsx [hunk 2]
```diff
@@ -44,47 +44,61 @@ class TeamsDataTable extends Component {
     );
   }
 
+  renderShowsTotal(start, to, total) {
```
[the remark this answers is not in the document]
Comment (ZmagoD) [reply app/javascript/src/scenes/SettingsPage/scenes/teams/components/TeamsDataTable.jsx]: this method can be static

File: app/javascript/src/scenes/SettingsPage/scenes/teams/index.jsx [hunk 3]
```diff
@@ -22,7 +22,7 @@ const Wrapper = styled.div`
 const SettingsTeams = ({ teams }) =>
   <Wrapper>
     <Breadcrumb>
-      <Breadcrumb.Item active={false}>
```
Comment (ZmagoD) [inline app/javascript/src/scenes/SettingsPage/scenes/teams/index.jsx]: Is OK to remove the false? Actually why is this even here if you don't want to mark the item as active you just remove the property

Comment (mlorb) [reply app/javascript/src/scenes/SettingsPage/scenes/teams/index.jsx]: It should be true, so short version of `active={true}` is just `active`. ESLint said so.

File: app/javascript/src/scenes/SettingsPage/scenes/teams/components/TeamsDataTable.jsx [hunk 4]
```diff
@@ -9,6 +9,21 @@ import DataTable from "../../../../../components/data_table";
 import { SETTINGS_TEAMS_ROUTE } from "../../../../../config/routes";
 
 class TeamsDataTable extends Component {
+  static renderShowsTotal(start, to, total) {
+    return (
+      <span>
+        <FormattedMessage
+          id="settings_page.shows_total_entries"
```
Comment (ZmagoD) [inline app/javascript/src/scenes/SettingsPage/scenes/teams/components/TeamsDataTable.jsx]: :+1:

**Pull request merged**